src/client.ts

```diff
@@ -159,7 +160,8 @@ class DecisionClient {
                 'in production environments.\n' +
                 '--------------------------------------------------------------'
             );
-            headers['x-adzerk-explain'] = additionalOpts.apiKey || '';
+            headers['x-adzerk-explain'] =
+              additionalOpts.apiKey || JSON.stringify(additionalOpts.desiredAd) || '';
```

Comment:
This still isn't going to work. There's no api key included if there's a desired ad. And there's still no support for desired ad map. Please look at numbers 2, 3, and 4 in https://app.shortcut.com/adzerk/story/23169/update-sdks-to-support-explainer-2-0-input-output for all of the supported JSON formats for the `x-adzerk-explain` header.

Reply:
Updated